homework_2/store/src/pages/user.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+export class User extends Component {
+    render() {
+        return (
+            <div className="container">
+                <h1>USER PAGE</h1>
+                <div className="row">
+                    <div className="col-8">
+                        <div className="row">
+                            {
+                                this.props.products.map(product => {
+                                    let inCart = product.inCart ? (
```

Comment:
also, you could compare product inside inCart state property instead of check property of item